File: LinearSearch.py
def linearsearch(arr, x):
    for i in range(len(arr)):
        if arr[i]==x:
            return i+1;
    return -1;

print (linearsearch([23,4,67,8,12,3],67))
print (linearsearch([23,4,67,8,12,3,45,62],9))

File: LinearSearch.java
/*
Linear Search algorithm implementation
Space complexity = O(1)
Time complexity = O(n)
*/
public class LinearSearch{

	private static int search(int arr[], int num){
		for (int i=0; i<arr.length ;i++ )
		{
			if (arr[i]==num)
			{
				return i+1;
			}
		}
		return -1;
	}

	public static void main(String[] args){
		int arr[] = {90,11,56,4,0,34,78,21,83,40};
		int searchNum = 34;

		System.out.printf("Number %d is at position %d in the given array.",searchNum, LinearSearch.search(arr, searchNum));
	}
}